src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -104,7 +104,7 @@ typedef enum {
     /* 11 */ ENKO_ANIM_PUNCHING_NOMORPH,
     /* 12 */ ENKO_ANIM_STANDING_HAND_ON_CHEST,
     /* 13 */ ENKO_ANIM_STANDING_HANDS_ON_HIPS,
-    /* 14 */ ENKO_ANIM_KNOWITALL_SEATED,
+    /* 14 */ ENKO_ANIM_SEATED,
     /* 15 */ ENKO_ANIM_SEATED_CROSSED_ARMS_LEGS,
```

Comment:
prob seated -> sitting (and in the anim too), but not a huge deal

Reply:
other "seated" anims too? (if you want, it makes literally no difference imo lol)

Reply:
id say so cause it seems like basically every other animation here is -ing